sdk/tests/js/tests/tex-input-validation.js

```diff
@@ -330,11 +330,11 @@ testCases = [
   { target: gl.TEXTURE_2D,
     format: 0x1903, // GL_RED
     type: gl.UNSIGNED_BYTE,
-    expectedError: gl.INVALID_ENUM },
+    expectedError: gl.INVALID_OPERATION },
```

Comment:
Same here, in ES2, RED is not a valid enum for format, so INAVLID_ENUM is correct.